src/sqltokenizer.rs

```diff
@@ -158,13 +162,20 @@ impl Tokenizer {
         let mut tokens: Vec<Token> = vec![];
 
         while let Some(token) = self.next_token(&mut peekable)? {
+
+            if token == Token::Whitespace('\n') {
+                self.line += 1;
+                self.col = 0;
+            } else if token == Token::Whitespace('\t') {
+                self.col += 1;
```

Comment:
so col should increase for each character in the SQL statement, and for `\t` I think it should increase by 4 (I know that's a bit arbitrary but I just checked in a text editor and that is the default there)